core/services/fluxmonitorv2/flux_monitor.go

```diff
@@ -788,30 +811,37 @@ func (fm *FluxMonitor) pollIfEligible(pollReq PollRequestType, deviationChecker
 	// Call the v2 pipeline to execute a new pipeline run
 	// Note: we expect the FM pipeline to scale the fetched answer by the same
 	// amount as precision
-	runID, answer, err := fm.pipelineRun.Execute(metaDataForBridge)
+	ctx := context.Background()
```

Comment:
context.TODO? Would we ever want to cancel this one?

Reply:
hm good question. I think the way we have it is ok now - it means that we wait for it to finish upon node shutdown